src/Pages/ProfilePage/index.tsx

```diff
@@ -1,3 +1,130 @@
-export default function ProfilePage() {
-  return <div>ProfilePage</div>;
-}
+import { useEffect, useState, useRef } from "react";
+import * as S from "./style";
+import { RequestList } from "../../components";
+import { ArrowButtonIcon } from "../../Assets/svg";
+import { UserInfo } from "../../types/user";
+import { Type } from "../../types/type";
+
+const data = {
+  id: 12,
+  username: "안철수",
+  grade: 1,
+  level: 13,
+  gender: "MALE",
+  type: "PORORO",
+  point: 0,
+  major: "FRONT",
+  requestList: [
+    {
+      requestId: 11,
+      title: "안녕하세요",
+      content: "아니 이거 맞아 내가 할말이 없다",
+      requestType: "TYPE",
+      authorName: "박미리",
+    },
+    {
+      requestId: 12,
+      title: "안녕하세요",
+      content: "아니 이거 맞아 내가 할말이 없다",
+      requestType: "STUDY",
+      authorName: "박미리",
+    },
+    {
+      requestId: 13,
+      title: "안녕하세요",
+      content: "아니 이거 맞아 내가 할말이 없다",
+      requestType: "TYPE",
+      authorName: "박미리",
+    },
+  ],
+};
+
+const typeList: Type = {
+  PORORO: "뽀로로",
+  LUPI: "루피",
+  POBI: "포비",
+  EDI: "에디",
+};
+
+const ProfilePage = () => {
+  const [isScrollable, setIsScrollable] = useState(false);
+  const [userInfo, setUserInfo] = useState<UserInfo>({
+    id: null,
+    username: null,
+    grade: null,
+    level: null,
+    gender: null,
+    type: null,
+    point: null,
+    major: null,
+    requestList: [],
+  });
+
+  const ref = useRef<HTMLDivElement>(null);
+  const gender = userInfo.gender === "MALE" ? "남자" : "여자";
+  const type = typeList[userInfo.type! as keyof Type];
+  const profileSrc = `src\\Assets\\png\\${userInfo.type}.png`;
+
+  useEffect(() => {
+    setUserInfo(data);
+  }, []);
+
+  const handleWindowResize = () =>
+    window.innerHeight < 1012 ? setIsScrollable(true) : setIsScrollable(false);
+
+  useEffect(() => {
+    window.addEventListener("resize", handleWindowResize);
+
+    return window.removeEventListener("resize", handleWindowResize);
```

Comment:
```suggestion
    return () => {
      window.removeEventListener("resize", handleWindowResize);
    };
```

클린업 함수는 이렇게 작성해야 작동할 것 같습니당

Reply:
아 제가 놓치고 있었네요 찾아주셔서 감사합니다!